macros/persist_pass_through_columns.sql

```diff
@@ -1,8 +1,8 @@
-{% macro persist_pass_through_columns(pass_through_variable) %}
+{% macro persist_pass_through_columns(pass_through_variable, identifier, transform='') %}
```

Comment:
This works, but I would feel more comfortable if we added a default value for `identifier`, just to be safe.

Reply:
added identifier=none